src/microbe_stage/PatchMapGenerator.cs

```diff
@@ -136,7 +150,8 @@ public static PatchMap Generate(WorldGenerationSettings settings, Species defaul
         };
         map.AddPatch(iceShelf);
 
-        var seafloor = new Patch("PATCH_PANGONIAN_SEAFLOOR", 10,
+        localizedName = GetPatchLocalizedName(name, "SEA_FLOOR").ToString();
```

Comment:
Converting this to string here will lose information. I think you should also modify the Patch class so that the name is a localized string in it. That way when game language is changed the "volcanic vents" part can correctly react to translations change.

This current approach also unnecessarily passes this string to the translation system so even if the above thing I mentioned is not done, that needs cleanup.